### PR TITLE
Fallback to n_jobs=1 if NWB Inspector fails

### PR DESCRIPTION
**Pull request by garrettmflynn**
This PR is a quick fix for https://github.com/NeurodataWithoutBorders/nwbinspector/issues/415 where the GUIDE will fall back to `n_jobs=1` if using multiple jobs fails.

### CODE REVIEW

File: pyflask/manageNeuroconv/manage_neuroconv.py [hunk 1]
```diff
@@ -593,18 +593,22 @@ def inspect_nwb_folder(payload):
...
+            "check_description",
+            "check_data_orientation",
+        ],  # TODO: remove when metadata control is exposed
+        config=load_config(filepath_or_keyword="dandi"),
+        **payload,
     )
 
+    try:
+        messages = list(inspect_all(**kwargs))
+    except:
```
Comment (CodyCBakerPhD) [inline pyflask/manageNeuroconv/manage_neuroconv.py:608]: Please don't use bare `except` - add a match to the exact type that occurs with the pickling problem if you can

Comment (garrettmflynn) [reply pyflask/manageNeuroconv/manage_neuroconv.py]: Updated to check for a `PicklingError`. Is it alright to bypass anything from pickle as an error with multithreading?

Comment (CodyCBakerPhD) [reply pyflask/manageNeuroconv/manage_neuroconv.py]: > Updated to check for a PicklingError. Is it alright to bypass anything from pickle as an error with multithreading?

Probably OK, but I always prefer to be as explicit as possible; you can add a second check `assert "some expected part of the specific message for this specific isssue" in str(e)` to be extra sure it's the problem you see

Comment (garrettmflynn) [reply pyflask/manageNeuroconv/manage_neuroconv.py]: Okay done.

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**